guides/installation.md

```diff
@@ -54,7 +54,7 @@ sudo swapon /swapfile
 
 You should now be able to run `amber` in the command line.
 
-#### macOS
+## MacOS
```

Comment:
I think it's official name is lowercase `macOS` but probably better capitalized here.

Reply:
ooops, I can change it to lowercase if you think it's better

Reply:
Yes please.  I'm just bikeshedding.  lol.